### ISSUE TITLE
[NPC] Warchief Rend Blackhand (UBRS)

### ISSUE THREAD

**Issue by Dranah** [WoWManiaUK/Redemption#1527]
Warchief Rend Blackhand (http://www.wowhead.com/npc=10429)

How it should work: Each time Nefarian does an emote a wave of adds should spawn until Rend himself comes on his drake, and once his drake is wounded enough, he becomes attackable himself.

Issue: 
1. Rend never becomes attackable. He attempts mounting his drake as he spawns, but instead his drake disappears and he himself stays idle, targetable but cannot hit him or engage him. (if you're quick enough, you can kill his drake atleast, but Rend himself stays still unattackable, making it impossible to finish the fight and open the doors.)
2. All but the first wave of adds are missing before the boss.

As said, the doors remain closed so i did not manage to finish the instance because of this.
Attempted to reset redo, hoping he doesn't bug, but it happened the exact same way again.

**Comment by tvshadowz** [comment]
Currently:

The mobs after the first wave are not spawning 

The Warchief phase starts but he dissapears

**Comment by captainloozer** [comment]
Hi Dranah, and thank you for another detailed report.

There are several things bugged with this encounter.   I have recorded an example of how to successfully complete this encounter with the current state:

https://youtu.be/H0nr65kcWSg

First, you'll see that only the first wave spawns correctly.  Every other wave is non-existent and the gate never opens.
Next you'll notice that when rend blackhand mounts his drake and comes out, I slowly dps him down to about 10% then remove my  weapon.  This is because if he is killed too quickly, rend will not dismount, this mount will die and you will not get to kill Rend.  If this happens, the encounter cannot complete and the door behind you leading to Drakkisath will never open, leaving you stranded in the dungeon.  In order to prevent that, you must slowly DPS Gyth down past 5% and wait for Rend do dismount.
The third thing I found was that Rend visually has about 32k health, but it's obviously more than that, as seen in the video.  Although still easy for me to kill as a level 80, it is an extremely high amount compared to what a few level 60s are expected to be able to do.  This should be looked at as well.

**Comment by fvicaria** [comment]
If the encounter works we can confirm.
The waves were removed on puporse

**Comment by captainloozer** [comment]
If that is the case, then the only thing I see wrong with the fight is having to be really careful about how fast you kill him or else the fight bugs and he doesn't spawn, leaving you trapped in the instance.  My video confirms though that it is possible to successfully complete the encounter if you know what to do.  If this is staying "as is", feel free to close.  @fvicaria

**Comment by Dranah** [comment]
@captainloozer 
I was running with a party of lvl 60-61s, and I took around 10-15sec to take down Gyth on the first try when Rend bugged.
The second try, I intended to slow dps, but Rend never even mounted up at all. His drake disappeared before i could touch it at all, 3sec after it spawned, none of them ever moved out from behind the gate.
What happened was, Gyth spawned, Rend ran next to Gyth, Gyth despawned and Rend just stood there, unattackable.

I don't particularly mind for the waves missing beforehand, I just reported that too if i was already here.
